### DISCUSSION TITLE
Update-Problem?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2690] [General]
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/2c3b60a5-ebdc-4716-b82d-8e4bd4de93ab)
Beim Update werden nur 2 Instanzen angezeigt, Instanz .1 fehlt, obwohl gestartet?
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/52b9585d-bf11-44c5-9d6f-86300e041832)

**Comment by Zefau** [comment]
Alles in Richtung Update Prozess obliegt ioBroker. Da bin ich raus.
